test/python/unit/api/test_datatype_api.py

```diff
@@ -209,6 +215,10 @@ def test_datatype_names(solver):
     with pytest.raises(RuntimeError):
         dt.getSelector("cons")
 
+    # possible to construct null datatype declarations if not using solver
```

Comment:
```suggestion
    # possible to construct null datatype declarations if not using mkDatatypeDecl
```

(I was confused at first because you are actually using `solver`)

Reply:
done